mne/time_frequency/tfr.py

```diff
@@ -3830,8 +3941,16 @@ def combine_tfr(all_tfr, weights="nave"):
 
     Notes
     -----
+    Aggregating multitaper TFR datasets with a taper dimension such as for complex or
+    phase data is not supported.
+
     .. versionadded:: 0.11.0
     """
+    if any("taper" in tfr._dims for tfr in all_tfr):
+        raise NotImplementedError(
+            "Aggregating multitaper tapers across TFR datasets is not supported."
+        )
+
```

Comment:
It's a similar case to averaging for the `time_frequency.combine_tfr()` function (which also gets called by the `grand_average()` function).

On a side note, I noticed that while a public function, `combine_tfr()` is not listed in the API (the equivalent `combine_evoked()` is). Is this an oversight or an intended omission?